protocols/gossipsub/src/config.rs

```diff
@@ -825,4 +835,81 @@ mod test {
 
         dbg!(builder);
     }
+
+    #[inline]
```

Comment:
```suggestion
```

Inlining does not always bring a performance win, e.g. the increased code size might actually hurt performance, and this is likely not a compile time win. What is the reasoning for suggesting to the compiler to inline this function? Does it really matter for testing code in the first place?